app/models/label/LabelValidationTable.scala

```diff
@@ -165,6 +167,31 @@ object LabelValidationTable {
     }
   }
 
+  /**
+   * Deletes a validation in the label_validation table. Also updates validation counts in the label table.
+   */
+  def deleteLabel(label: LabelValidation): Int = db.withTransaction { implicit session =>
```

Comment:
This should be renamed to `deleteValidation`, you aren't deleting a label here! Or `deleteLabelValidation`